packages/playwright/src/screenplay/abilities/BrowseTheWeb.ts

```diff
@@ -0,0 +1,563 @@
+import { Ability, LogicError, UsesAbilities } from '@serenity-js/core';
+import {
+    Browser,
+    BrowserContext,
+    BrowserType,
+    ElementHandle,
+    Frame,
+    LaunchOptions,
+    Mouse,
+    Page,
+    Response,
+} from 'playwright';
+import { PageFunction } from 'playwright/types/structs';
+
+import { Stack } from '../../utils';
+import { NavigateOptions } from '../interactions';
+import { ScreenshotOptions } from '../options/screenshotOptions';
+
+/**
+ * @desc
+ *  An {@link @serenity-js/core/lib/screenplay~Ability} that enables the {@link Actor} to interact with web front-ends using {@link protractor}.
+ *
+ * @example <caption>Using the protractor.browser</caption>
+ *  import { Actor } from '@serenity-js/core';
+ *  import { BrowseTheWeb, Navigate, Target } from 'serenityjs-playwright';
+ *  import { Ensure, equals } from '@serenity-js/assertions';
+ *  import { chromium } from 'playwright';
+ *
+ *  const actor = Actor.named('Wendy').whoCan(
+ *      BrowseTheWeb.using(chromium),
+ *  );
+ *
+ *  const HomePage = {
+ *      Title: Target.the('title').located(by.css('h1')),
+ *  };
+ *
+ *  actor.attemptsTo(
+ *      Navigate.to(`https://serenity-js.org`),
+ *      Ensure.that(Text.of(HomePage.Title), equals('Serenity/JS')),
+ *  );
+ *
+ * @see https://www.protractortest.org/
+ *
+ * @public
+ * @implements {@serenity-js/core/lib/screenplay~Ability}
+ * @see {@link @serenity-js/core/lib/screenplay/actor~Actor}
+ */
+export class BrowseTheWeb implements Ability {
+    private static actorMap = new Map();
+
+    /**
+   * @private
+   */
+    private _browseContext: BrowserContext;
+
+    /**
+   * @private
+   */
+    private _browser: Browser;
+    /**
+   * @private
+   */
+    private launchOptions?: LaunchOptions;
+
+    /**
+   * @private
+   */
+    private _page: Page;
+
+    /**
+   * @private
+   */
+    private _workingContext: Page | Frame;
+
+    /**
+   * @private
+   */
+    private storedContext: Stack<Page | Frame>;
+
+    protected async workingContext(): Promise<Page | Frame> {
+        if (!this._workingContext) {
+            this._workingContext = await this.page();
+        }
+        return this._workingContext;
+    }
+
+    protected setWorkingContext(newContext: Page | Frame): void {
+        this._workingContext = newContext;
+    }
+
+    /**
+   * @private
+   * @desc
+   *  Playwright browser context. It's recommended by playwright to use context as wrappers around pages
+   *  for frameworks
+   */
+    private async browser(): Promise<Browser> {
```

Comment:
If you'd like lazy loading then `BrowseTheWeb` could implement [`Initialisable`](https://github.com/serenity-js/serenity-js/blob/master/packages/core/src/screenplay/abilities/Initialisable.ts) (and potentially [`Discardable`](https://github.com/serenity-js/serenity-js/blob/master/packages/core/src/screenplay/abilities/Discardable.ts) if we want to kill the browser afterwards)